license-exceptions/CNCF-licensing-exceptions.csv

```diff
@@ -0,0 +1,450 @@
+jquery,MIT,allowlisted,2019-11-01,
+jQuery Slideto,MIT,contained directly in repo; not auto-allowlist because: Not isolated; approved by GB exception 2019-03-11,2019-11-01,
+jQuery Wiggle,MIT,contained directly in repo; not auto-allowlist because: Not isolated; approved by GB exception 2019-03-11,2019-11-01,
+jquery.scrollto,MIT,allowlisted,2019-11-01,
+marked,MIT,contained directly in repo; not auto-allowlist because: Not isolated; approved by GB exception 2019-03-11,2019-11-01,
+mime,MIT,allowlisted,2019-11-01,
+minimist,MIT,allowlisted,2019-11-01,
+moment.js,MIT,contained directly in repo; not auto-allowlist because: Not isolated; approved by GB exception 2019-03-11,2019-11-01,
+node-static,MIT,allowlisted,2019-11-01,
+normalize.css,MIT,contained directly in repo; not auto-allowlist because: Not isolated; approved by GB exception 2019-03-11,2019-11-01,
+Octicons,MIT,contained directly in repo; not auto-allowlist because: Not isolated; approved by GB exception 2019-03-11,2019-11-01,
+optimist,MIT,allowlisted,2019-11-01,
+parseUri,MIT,contained directly in repo; not auto-allowlist because: Not isolated; approved by GB exception 2019-03-11,2019-11-01,
+Pause,MIT,not auto-allowlist because: No measure of stars / forks; approved by GB exception 2019-03-11,2019-11-01,
+ui-router for Angular,MIT,allowlisted,2019-11-01,
+Underscore.js,MIT,contained directly in repo; not auto-allowlist because: Not isolated; approved by GB exception 2019-03-11,2019-11-01,
+vbom.ml/util,MIT,allowlisted,2019-11-01,
+wordwrap,MIT,allowlisted,2019-11-01,
+github.com/c4milo/gotoolkit,MPL-2.0,not auto-allowlist because: Non-allowlist license(s); approved by GB exception 2019-03-11,2019-11-01,
+github.com/d2g/dhcp4client,MPL-2.0,not auto-allowlist because: Non-allowlist license(s); approved by GB exception 2019-03-11,2019-11-01,
+github.com/go-sql-driver/mysql,MPL-2.0,not auto-allowlist because: Non-allowlist license(s); approved by GB exception 2019-03-11,2019-11-01,
+github.com/hashicorp/errwrap,MPL-2.0,not auto-allowlist because: Non-allowlist license(s); approved by GB exception 2019-03-11,2019-11-01,
+github.com/hashicorp/go-cleanhttp,MPL-2.0,not auto-allowlist because: Non-allowlist license(s); approved by GB exception 2019-03-11,2019-11-01,
+github.com/hashicorp/go-multierror,MPL-2.0,not auto-allowlist because: Non-allowlist license(s); approved by GB exception 2019-03-11,2019-11-01,
+github.com/hashicorp/golang-lru,MPL-2.0,not auto-allowlist because: Non-allowlist license(s); approved by GB exception 2019-03-11,2019-11-01,
+github.com/hashicorp/hcl,MPL-2.0,not auto-allowlist because: Non-allowlist license(s); approved by GB exception 2019-03-11,2019-11-01,
+github.com/hooklift/iso9660,MPL-2.0,not auto-allowlist because: Non-allowlist license(s); approved by GB exception 2019-03-11,2019-11-01,
+defusedxml,Python-2.0,allowlisted,2019-11-01,
+go-retryablehttp,MPL-2.0,not auto-allowlist because: Non-allowlist license(s); approved by GB exception 2021-07-19,2021-07-19,

```

Comment:
should this be `github.com/hashicorp/go-retryablehttp`?
same for other hashicorp modules below which are either missing a `github.com/hashicorp/` prefix or `github.com/` prefix